packages/@n8n/api-types/src/dto/credentials/credentials-get-one-request.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { z } from 'zod';
+import { Z } from 'zod-class';
+
+export class CredentialsGetOneRequest extends Z.class({
```

Comment:
Since these are the query params, maybe make it explicit?
```suggestion
export class CredentialsGetOneRequestQuery extends Z.class({
```